README.md

```diff
@@ -1,226 +1,188 @@
-address = sdk.get_address()
+from kin import Environment
+
+MY_CUSTOM_ENVIRONMENT = Environemnt('name','horizon endpoint','network passphrase','kin issuer','friendbot url'(optional))
 ```
 
-### Getting Account Balance
+Once you have a KinClient, you can use it to get a KinAccount object: 
 ```python
-# Get native (lumen) balance of the SDK wallet
-native_balance = sdk.get_native_balance()
+# The KinAccount object can be initizlied in a number of ways:
 
-# Get KIN balance of the SDK wallet
-kin_balance = sdk.get_kin_balance()
+# With a single seed:
+account = client.kin_account('seed')
 
-# Get native (lumen) balance of some account
-native_balance = sdk.get_account_native_balance('address')
+# With specific channels:
+account = client.kin_account('seed', channel_secret_keys=['seed1','seed2','seed3'...])
 
-# Get KIN balance of some account
-kin_balance = sdk.get_account_kin_balance('address')
+# With deterministic channels
+# This will generate the channels secret seeds for you, based on your primary seed. 
+# The generated channels will always be the same for the same primary seed.
+# If this is your first time using these channels, you should set 'create_channels' to true.
+# This will create the channel account from your primary seed (and will cost 1.6 * number of channels XLM)
+account = client.kin_account('seed',channels=number, create_channels=True/False)
+
+# Every seed can only send one transaction per ledger (~5 seconds),
+# so using channels will greatly increase your concurrency.
 ```
 
-### Getting Account Data
+## Client Usage
+Most methods provided by the KinClient to query the blockchain about a specific account, can also be used from the KinAccount object to query the blockchain about itself
+
+### Getting Account Balance
 ```python
-# returns kin.AccountData
-account_data = sdk.get_account_data('address')
+# Get KIN/XLM balance
+balances = client.get_account_balances('address')
+kin_balance = balances['KIN']
+xlm_balance = balances['XLM']
 ```
 
-### Checking If Account Exists
+### Getting Account Data
 ```python
-account_exists = sdk.check_account_exists('address')
+account_data = client.get_account_data('address')
 ```
 
-### Creating a New Account
+### Checking Account Status
 ```python
-# create a new account prefunded with MIN_ACCOUNT_BALANCE lumens
-tx_hash = sdk.create_account('address')
+from kin import AccountStatuses
+
+status = client.get_account_status('address')
 
-# create a new account prefunded with a specified amount of native currency (lumens).
-tx_hash = sdk.create_account('address', starting_balance=1000)
+# status can be one of:
+AccountStatuses.NOT_CREATED
+AccountStatuses.NOT_ACTIVATED
+AccountStatuses.ACTIVATED
 ```
 
-### Checking if Account is Activated (Trustline established)
+
+### Getting Transaction Data
 ```python
-# check if KIN is trusted by some account
-kin_trusted = sdk.check_account_activated('address')
+# Get information about a specific transaction
```

Comment:
I think that all these comments should be extracted from code to markdown. moreover, the numbered list should be converted to a list/bullets, it will be much readable this way